packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -146,8 +147,7 @@ export function Initialization(options?: {
       }
 
       if (!this.bindings) {
-        // TODO: add optional renderLoad() method to render placeholders
-        return `${getElement(this).nodeName.toLowerCase()}_loading`;
+        return;
```

Comment:
Yeah.... Finally, I think it doesn't really make sense to render anything until the bindings aren't passed down, which is supposed to be quite fast. Loading state makes more sense to display until the results are fetching, so only result dependent components would require a renderLoading method.

Reply:
Hmm, one of the goals of loading animations is to give end-users feedback that the page is loading compared to looking at a white screen. This may be fast for us, but can take considerable time when on a weaker device and a 2G connection (we'd need to test). I think it's too early to look into this, we can open a jira to come back to in the future,